microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -25,7 +25,11 @@ If you don't have an account or if you have an existing commercial account with
 If you have a developer account registered on Partner center, use the corresponding Microsoft account \(MSA\) to sign into your developer account, and then enroll in the Microsoft Edge program.  
 
 > [!NOTE]
-> Today the Microsoft Edge extensions does not support signing up with a work or school account.  In the future, the Microsoft Edge extensions team plans to support linking Azure AD tenants to MSA accounts for extension management.  
+>  You can choose to link your work account on Partner center to manage your Edge extensions. For this, you first need to register to Partner center using a Microsoft account. To link your work account, navigate to **Tenants** under **Account settings** and select either **Associate your Azure AD account with Partner center** or **Create a new Azure AD**.
+
+If you are enrolled to some other program on Partner center, login with the Admin credentials of your work account and enroll into the Edge program using the same credentials.
```

Comment:
We can add some more details here on how to find the Admin credentials for your current work account.